source/SUBMIT.rst

```diff
@@ -289,9 +287,18 @@ you can easily test your preprint build locally.
 
 **Step 2 - Data**
 
-If your executable content depends on a dataset, please make sure that your notebooks have access to it.
+First, make sure that your data is available online.
+
+You can now install `Repo2Data <https://github.com/SIMEXP/Repo2Data>`_, and configure the ``data_requirement.json`` with ``"dst": "./data"``.
+Navigate into your repo and run repo2data, all your data should appear in the ``data`` folder at the root of your repository.
```

Comment:
all your data should appear in the ``data`` folder at the root of your repository. --> your data will download to the ``data`` folder.